sdcm/tester.py

```diff
@@ -1364,20 +1364,23 @@ def get_cluster_baremetal(self):
             private_ips=self.params.get('db_nodes_private_ip'),
             user_prefix=self.params.get('user_prefix'),
             credentials=self.credentials,
+            ssh_username='ubuntu',
```

Comment:
I think better to store ssh_username in config files. May add new parameters `baremetal_db_ssh_username`, `baremetal_loader_ssh_username` or similar. Because it will be also need in future the ssh key file, which should be stored in aws key store

Reply:
I don't understand what's wrong with the configuration option we have today for the db user, why can't we use that ?

Why have other defaults in the code, or why introduce new configuration options ?